config

```diff
@@ -11,7 +11,7 @@ check_docker_version() {
         [ -z "$version" ] && echo can not retrieve docker version && return 1
         local major_version; major_version=$(echo "$version" | cut -d '.' -f 1)
         local minor_version; minor_version=$(echo "$version" | cut -d '.' -f 2)
-        [ "$major_version" -ne "$DOCKER_MAJOR_VERSION" ] && echo docker major version should be = $DOCKER_MAJOR_VERSION && return 1
+        [ "$major_version" -lt "$DOCKER_MAJOR_VERSION" ] && echo docker major version should be = $DOCKER_MAJOR_VERSION && return 1
```

Comment:
"should be <" 
du coup la on veut plus que ca qoit égal, mais bien inférieur strict

Reply:
Si si c est vraiment >=
Dans notre cas, c'est la version majeure 17